src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -0,0 +1,221 @@
+import CheckboxWithLabel from '../../components/CheckboxWithLabel';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import exampleCheckImage from '../../../assets/images/example-check-image.png';
+import Text from '../../components/Text';
+import {setupWithdrawalAccount} from '../../libs/actions/BankAccounts';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+class BankAccountStep extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.toggleTerms = this.toggleTerms.bind(this);
+        this.addManualAccount = this.addManualAccount.bind(this);
+        this.addPlaidAccount = this.addPlaidAccount.bind(this);
+
+        this.state = {
+            // One of CONST.BANK_ACCOUNT.SETUP_TYPE
+            bankAccountAddMethod: undefined,
+            hasAcceptedTerms: true,
+            routingNumber: '',
+            accountNumber: '',
+        };
+    }
+
+    toggleTerms() {
+        this.setState(prevState => ({
+            hasAcceptedTerms: !prevState.hasAcceptedTerms,
+        }));
+    }
+
+    /**
+     * @returns {Boolean}
+     */
+    canSubmitManually() {
+        return this.state.hasAcceptedTerms
+
+            // These are taken from BankCountry.js in Web-Secure
+            && CONST.BANK_ACCOUNT.REGEX.IBAN.test(this.state.accountNumber.trim())
+            && CONST.BANK_ACCOUNT.REGEX.SWIFT_BIC.test(this.state.routingNumber.trim());
+    }
+
+    addManualAccount() {
+        setupWithdrawalAccount({
+            acceptTerms: this.state.hasAcceptedTerms,
+            accountNumber: this.state.accountNumber,
+            routingNumber: this.state.routingNumber,
+            setupType: CONST.BANK_ACCOUNT.SETUP_TYPE.MANUAL,
+
+            // Note: These are hardcoded as we're not supporting AU bank accounts for the free plan
+            country: CONST.COUNTRY.US,
+            currency: CONST.CURRENCY.USD,
+            fieldsType: CONST.BANK_ACCOUNT.FIELDS_TYPE.LOCAL,
+        });
+    }
+
+    /**
+     * @param {Object} params
+     * @param {String} params.password
+     * @param {Object} params.account
+     * @param {String} params.account.bankName
+     * @param {Boolean} params.account.isSavings
+     * @param {String} params.account.addressName
+     * @param {String} params.account.ownershipType
+     * @param {String} params.account.accountNumber
+     * @param {String} params.account.routingNumber
+     * @param {String} params.account.plaidAccountID
+     */
+    addPlaidAccount(params) {
+        setupWithdrawalAccount({
+            acceptTerms: true,
+            setupType: CONST.BANK_ACCOUNT.SETUP_TYPE.PLAID,
+
+            // Note: These are hardcoded as we're not supporting AU bank accounts for the free plan
+            country: CONST.COUNTRY.US,
+            currency: CONST.CURRENCY.USD,
+            fieldsType: CONST.BANK_ACCOUNT.FIELDS_TYPE.LOCAL,
+            password: params.password,
```

Comment:
I think the `password` field and below are _not_ hard-coded, so maybe they should be grouped separately from the hard-coded values.

Reply:
Ah yes, I see what you mean there.